srcpkgs/wmx/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'wmx'
 pkgname=wmx
 version=8
-revision=2
+revision=3
 build_style=gnu-configure
+make_build_args="LDFLAGS+=-lXft LDFLAGS+=-lfontconfig
+ CXXFLAGS+=-I${XBPS_CROSS_BASE}/usr/include/freetype2"
 makedepends="libX11-devel libXext-devel libXmu-devel libXpm-devel freetype-devel
  libXft-devel fontconfig-devel libXcomposite-devel"
-short_desc="a simple window manager for X"
+short_desc="Simple window manager for X"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="BSD"
```

Comment:
There is no license nor copyright headers, this package seems to be all rights reserved.